### ISSUE TITLE
parameters which refer to global properties aren't updated when properties change

### ISSUE THREAD

**Issue by meetar** [tangrams/tangram#390]
- Set a global property: `global: { lines: true }`
- Reference it in a draw style:
```yaml
layers:
    earth:
        data: { source: osm }
        draw:
            lines:
                visible: global.lines
```
- Change the value of the property at run-time: `scene.config.global.lines = false;`
- Update with `scene.rebuild()`
- Check the value of the referenced value: `scene.config.earth.draw.lines.visible`

**RESULT:**

The referenced value does not change.

**EXPECTED RESULT:**

The referenced value will match the referred value.

**Comment by meetar** [comment]
occurs in all cases

**Comment by bcamper** [comment]
WIP fix in https://github.com/tangrams/tangram/compare/reapply-globals

**Comment by bcamper** [comment]
Released in v0.10.1 and 0.10.2
